one_fm/overrides/leave_application.py

```diff
@@ -715,5 +729,15 @@ def reassign(self):
                 self.reassign_operation_settings(settings=value)
 
 
+def get_status_in_arabic(status):
```

Comment:
In order to make this reusable, you can add this as a child table in the onefm general settings, so that other parts of the system can use it, also, there could be a correction or a change in the nearest future

Reply:
How can I do that

Reply:
@yusuffAFEX  By adding a child table to the one fm general settings doctype, so those needed fields and their arabic equivalent, would be added, and wherever it is needed in the code, it will be fetched.

![image](https://github.com/user-attachments/assets/bd89ffca-e924-4129-886d-557577a52ac9)
The table would look like this, but instead of one coulmn, there would be two, the english equivalent and the arabic equivalent

Reply:
@yusuffAFEX , kindly take note

Reply:
Yeah, I have done the needful